python/README.md

```diff
@@ -221,6 +221,31 @@ $ poetry run python main.py rewrite '{"text": "In the cycling world, power meter
 ]
 ```
 
+### Text Completion
+
+Text Completion is called within the editor pane. It allows a user to trigger the LLM to write the next block of text in the document.
+
+It takes an input of text with additional optional parameters and returns a list of rewrite `choices`.
```

Comment:
I think we can document the optional parameters, and their defaults.

Reply:
IMO documenting that here will likely lead to documentation rot when the optional params are inevitably changed in `typing.py` or `rewrite.py` but not in the readme. In hindsight, I think I overdid this readme.

I can capture the defaults in a docstring somewhere as a compromise.

Reply:
Fair enough.